src/main/common/filter.h

```diff
@@ -45,6 +47,13 @@ typedef struct biquadFilter_s {
     float x1, x2, y1, y2;
 } biquadFilter_t;
 
+#define BIQUAD_LPF_ORDER_MAX 6
+
+typedef struct biquadFilterCascade_s {
+    biquadFilter_t biquad[(BIQUAD_LPF_ORDER_MAX + 1) / 2];   // each section is of second order
+    int sections;
```

Comment:
sections should be first in structure - that way biquad array can be any length ...

Reply:
cool - that makes sense. handled.